lib/models/release.coffee

```diff
@@ -55,21 +56,46 @@ getReleaseModel = (deps, opts) ->
 	# });
 	#
 	# @example
+	# balena.models.release.get('7cf02a6').then(function(release) {
+	#		console.log(release);
+	# });
+	#
+	# @example
 	# balena.models.release.get(123, function(error, release) {
 	#		if (error) throw error;
 	#		console.log(release);
 	# });
 	###
-	exports.get = (id, options = {}, callback) ->
+	exports.get = (commitOrId, options = {}, callback) ->
 		callback = findCallback(arguments)
 
-		return pine.get
-			resource: 'release'
-			id: id
-			options: mergePineOptions({}, options)
-		.tap (release) ->
-			if not release?
-				throw new errors.BalenaReleaseNotFound(id)
+		return Promise.try ->
+			if not commitOrId?
+				throw new errors.BalenaReleaseNotFound(commitOrId)
+
+			if isId(commitOrId)
```

Comment:
Are "short" commit hashes acceptable in "commitOrId"? Because commit hashes are hex strings, and it's not "that rare" for a 7-char hex string to be composed of decimal digits and thus return true as `isId`.

Reply:
@pdcastro `isId` is using `_.isNumber` which returns false for strings, so an all numeric short commit hash will not be used as a release id.
I still see your concern, but I will make sure to handle this in the CLI (next Friday probably).

Reply:
Cool 👍